integration-tests/docker/docker-compose.base.yml

```diff
@@ -356,3 +356,20 @@ services:
     env_file:
       - ./environment-configs/common
       - ./environment-configs/router-custom-check-tls
+
+  druid-openldap:
+    image: osixia/openldap:1.4.0
```

Comment:
This uses MIT license. Do we need to add this to any license file or anything like that?

Reply:
Since this is only pulled while running the integration tests, I think it can be omitted, since it won't actually ever be included in a source/binary/docker distribution, which is what the license and notice files cover.